client/src/api/connectJSON.js

```diff
@@ -61,3 +61,45 @@ export async function deleteWholeWeek() {
     method: "DELETE",
   });
 }
+
+export async function postGroceryList() {
+  const response = await fetch("api/shoppingList/merge");
```

Comment:
Make sure to fetch from `/api`, otherwise links might be invalid if you are on a subpage. 
```suggestion
  const response = await fetch("/api/shoppingList/merge");
```